ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-upgrade-from-stable-4.16.yaml

```diff
@@ -402,8 +611,25 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-azure-ipi-publish-mixed-ingress-internal
-- as: azure-ipi-oidc-arm-f14
-  cron: 42 23 3,17 * *
+- as: azure-ipi-mto-arm-mixarch-f28
```

Comment:
@lwan-wanglin do we need to cover the install profile in upgrade ci ?

Reply:
Seems @lwan-wanglin is on PTO, can we merge it first and remove it when we it indeed not need it?

Reply:
I prefer to not add it for now, after confirmed that we need to cover it, then add it later.

Reply:
Sure, removed